db/seeds.rb

```diff
@@ -45,3 +45,44 @@
   category_id: 2,
 )
 
+Video.create!(title: 'Big Bang Theory 2',
+  description: "Mensa-fied best friends and roommates Leonard and Sheldon, physicists who work at the California Institute of Technology, may be able to tell everybody more than they want to know about quantum physics, but getting through most basic social situations, especially ones involving women, totally baffles them.",
+  small_cover_url: '/tmp/big_bang_sm.jpg', 
+  large_cover_url: '/tmp/big_bang_lg.jpg',
+  category_id: 1,
```

Comment:
Rather than hard code your category ids, you should assign the created categories above to variables and then use those in the associations. Database engines do not make any guarantees that the id column will start at 1 and be sequential, although that is generally the case.